src/CliqStateMachineUtils.jl

```diff
@@ -900,6 +900,7 @@ function determineCliqVariableDownSequence(subfg::AbstractDFG, cliq::TreeClique;
 
   #TODO don't use this getAdjacencyMatrixSymbols
   adj = DFG.getAdjacencyMatrixSymbols(subfg, solvable=solvable)
+  # adjTEST = DFG.getBiadjacencyMatrix(subfg, solvable=solvable)
```

Comment:
just as a breadcrumb, it returns adj, varLabels, FactorLabels. So adj [1, :] is now varLabels, subAdj is adj, but a sparse matrix

Reply:
this really helped thanks!!